src/codegen/doc/gi_docgen.rs

```diff
@@ -196,6 +196,36 @@ fn ns_type_to_doc(namespace: &Option<String>, type_: &str) -> String {
     }
 }
 
+fn find_virtual_method(
```

Comment:
```suggestion
fn find_virtual_method_by_name(
```

for consistency with the function just below?